index.js

```diff
@@ -35,12 +53,20 @@ function objectState(_initial) {
     emitState()
   }
 
-  function get(keypath) {
+  function callEmit (force) {
```

Comment:
pretty nitty, but this logic could just as easily live inside `emitState(force)`. Since that function already makes decisions about calling actuallyEmit immediately or not.